lib/travis/build/script/julia.rb

```diff
@@ -70,7 +70,11 @@ def setup
             when 'windows'
               sh.cmd %Q{curl -A "$CURL_USER_AGENT" -s -L --retry 7 -o julia-installer.exe '#{julia_url}'}
               sh.cmd 'chmod +x julia-installer.exe'
-              sh.cmd %Q{powershell -c 'Start-Process -FilePath julia-installer.exe -ArgumentList "/S /D=C:\\julia" -NoNewWindow -Wait'}
+              if Gem::Version.new(config[:julia]) >= Gem::Version.new('1.4') || config[:julia] == 'nightly'
```

Comment:
What happens if you pass `nightly` to `Gem::Version.new`?

Reply:
(errors)  malformed string

Reply:
Then won't this cause an error for Windows with Julia nightly?

Reply:
Yes I think you're right. I think I'll just have to switch around the short circuiting .